docs/src/quickstart-agones-xonotic.md

```diff
@@ -112,7 +112,7 @@ replace the `${GAMESERVER_IP}` and `${GAMESERVER_PORT}` values in your copy of `
 Run this configuration locally as:
 
 ```shell
-quilkin run -c ./client-compress.yaml`
+quilkin -c ./client-compress.yaml run`
```

Comment:
The reordering of this feels counter intuitive to me (thinking `docker run` vs `docker build` determining which args are able to be passed in).

I assume this is something we can't get away from?

@luna-duclos what are your thoughts?

Reply:
I am quite familiar to this in various tools, so its fine by me at least to have some top level configs

Reply:
Fair enough - I'm out voted 😄 I rescind my objections!